routes/index.js

```diff
@@ -22,7 +23,16 @@ function makeHandlerAwareOfAsyncErrors(handler) {
     try {
       await handler(req, res, next);
     } catch (error) {
-      next(error);
+      console.log(error);
+      if (error instanceof FcError) {
+        res.status(400).send({
+          code: error.code,
+          msg: error.msg,
+          data: error.data,
+        });
```

Comment:
should it be like this?

```suggestion
        res.status(400).send({
          error: error.code,
          error_msg: error.msg,
          data: error.data,
        });
```
cc @kc97ble

Reply:
it's too late :( Em sửa một đống file rối mới đọc được comment này :(